components/bl-stopwatch/README.md

```diff
@@ -24,14 +24,10 @@ This is a component for Backendless [UI-Builder](https://backendless.com/develop
 
 ## Styles
 
-**Dimensions**
-
 ```
 @bl-customComponent-stopwatch-font-size: 16px;
-```
-
-**Color**
-
-```
 @bl-customComponent-stopwatch-text-color: @appTextColor;
+@bl-customComponent-stopwatch-font-family: "Lucida Console", monospace;
 ```
+
+> **_NOTE:_** If you notice that the stopwatch twitches (this is especially noticeable when the component is centered horizontally), you may need to specify a font from the monospace family
```

Comment:
That does not look like a solution. Is it possible to define the char width by ourselves?

Reply:
@v-excelsior  I did not find a better option. Except to wrap each number in an element and assign it a width. I got the idea [here](https://stackoverflow.com/a/10149379).

Reply:
@StasivPavlo Since we know how many chars we have, wrapping each element into a separate span looks not so bad to me. Do you see any cons here?

Reply:
No, I will try to do it